blogs/2021/06/10/remote-repositories.md

```diff
@@ -141,7 +141,7 @@ Click the remote indicator in the lower left, then select **Continue Working on.
 You'll be presented with three options:
 
 * **Clone Repository Locally**: Clone the current repo to your local machine. A local file explorer will pop up, allowing you to select a location on disk to clone the remote repo.
-* **Clone Repository in Container Volume**: Clone the current repo in a Docker [container volume](https://code.visualstudio.com/docs/remote/containers#_quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume) using the Remote – Containers extension (you'll need the [Remote – Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension and Docker installed). VS Code will reload and connect using Remote – Containers, and the remote indicator will now read **Dev Container: {image name}**.
+* **Clone Repository in Container Volume**: Clone the current repo in a Docker [container volume](/docs/devcontainers/containers.md#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume) using the Remote – Containers extension (you'll need the [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension and Docker installed). VS Code will reload and connect using Remote – Containers, and the remote indicator will now read **Dev Container: {image name}**.
```

Comment:
```suggestion
* **Clone Repository in Container Volume**: Clone the current repo in a Docker [container volume](/docs/devcontainers/containers.md#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume) using the Dev Containers extension (you'll need the [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension and Docker installed). VS Code will reload and connect using Dev Containers, and the remote indicator will now read **Dev Container: {image name}**.
```

Reply:
Grrr, how can there still be reference to Remote - Containers? Darn, that pesky special dash character